semgrep/tests/performance/test_semgrep_rules_repo_perf.py

```diff
@@ -0,0 +1,198 @@
+import subprocess
+from pathlib import Path
+
+import pytest
+
+# output of `fd ya?ml$`
+# not calculating dynamically to avoid slow test collection on each pytest run
+RULE_PATHS: str = """
```

Comment:
Does this break if the file paths change in `semgrep-rules`? I know Grayson was recently working on changing the paths. This may also be a source of fragility in the future if the paths continue to evolve.

Reply:
Doesn't break, cause we're pinning to a specific SHA of the semgrep-rules repo. The purpose of these benchmarks is to test semgrep's performance — I didn't want to snapshot performance increases that come from optimized rule files. I expect not to update the SHA and this list for like 6 months.

I've updated the SHA and path list now though with @minusworld's file renames.